openfecwebapp/templates/macros/missing.html

```diff
@@ -1,14 +1,15 @@
 {% macro missing_financials(entity, cycle) %}
-<div class="message message--no-icon">
-  <h2 class="message__title" id="section-1-header" tabindex="0">No results</h2>
-  <p>Sorry, we didn't find any financial data for <span class="t-bold">{{ entity }}</span> in the <span class="t-bold">{{ cycle | fmt_year_range }}</span> election cycle.</p>
+<div class="message message--info">
+  <p>We don't have any financial data for <span class="t-bold">{{ entity }}</span> in <span class="t-bold">{{ cycle | fmt_year_range }}</span>.</p>
+  <p>This can either be because:</p>
```

Comment:
Now that I'm looking at this again, I'm worried that the "either" makes it seem like these are the only two possible reasons, which I don't think is the case. What if we said 

_Sometimes this happens when a candidate [or committee]:_ 

instead?